src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java

```diff
@@ -1093,8 +1094,11 @@ public AnalysisTreeResponse getNewickTree(@PathVariable Long submissionId, Local
 					}
 				}
 			} catch (NoSuchFileException e) {
-				logger.debug("File was not found: " + e.toString());
+				logger.error("File was not found: " + e.toString());
+			} catch (IOException e) {
+				logger.error("Unable to read input stream for file");
```

Comment:
Add the throwable cause here in the error logging so we can see the stack trace please.